policyengine_us/parameters/gov/states/id/tax/income/deductions/divisor.yaml

```diff
@@ -0,0 +1,15 @@
+description: Idaho divides the total deductions by this amount.
+values:
+  2021-01-01: 3_417
+metadata:
+  unit: currency-USD
+  label: Idaho deductions devisor
+  reference:
+    - title: Itemized Deduction Worksheet for Idaho
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EIS00407/EIS00407_01-05-2023.pdf
+    - title: TITLE 63 REVENUE AND TAXATION CHAPTER 30 INCOME TAX
```

Comment:
need chapter of income tax where the paragraph can be found

Reply:
can't find the paragraph of legislation code for the divisor